jablib/src/test/java/org/jabref/logic/importer/fetcher/transformers/ScopusQueryTransformerTest.java

```diff
@@ -0,0 +1,92 @@
+package org.jabref.logic.importer.fetcher.transformers;
+
+import java.util.Optional;
+
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.search.query.SearchQueryNode;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ScopusQueryTransformerTest {
```

Comment:
AI generates these kinds of tests and not `@ParameterizedTest`. 

I think, this tests does not bring in any value, does it? - Remove it.

Alternatively, I think, you can base on InfixTransformerTest. But I currently think, this is too much effort.